configuration/config-txt/video.md

```diff
@@ -336,6 +336,38 @@ Note that this simply **creates** the mode (group 2 mode 87). In order to make t
 
 This may not work if your monitor does not support standard CVT timings.
 
+### LCD display/touchscreen options
+
+#### ignore_lcd
+
+By default no DSI LCD detection is run (`ignore_lcd=1`). To enable detection (and accordingly the LCD display), use `ignore_lcd=0`
```

Comment:
By default the DSI LCD is used when detected over I2C.
``ignore_lcd=1`` will skip detection and not use the LCD.